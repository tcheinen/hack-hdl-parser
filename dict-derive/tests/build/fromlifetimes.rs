use dict_derive::FromPyObject;

#[derive(FromPyObject)]
struct Derived<'a, 'b> {
    a: &'a str,
    b: &'b str,
}

fn main() {}
