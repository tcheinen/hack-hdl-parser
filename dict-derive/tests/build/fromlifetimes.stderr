error: Deriving structs with more than one lifetime is not supported
 --> $DIR/fromlifetimes.rs:4:15
  |
4 | struct Derived<'a, 'b> {
  |               ^^^^^^^^
